Models/DTO/Administrator/GetDataTDTO.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Newtonsoft.Json;
+
+namespace API.Models.DTO.Administrator
+{
+    public class GetDataTDTO<T>
```

Comment:
I think `GetDataDTO<T>` would be more proper name here.

Reply:
No becauce there are two types DetDataDTO IEnumerable and simple T.